sdk/typescript/src/types/transactions.ts

```diff
@@ -339,6 +339,12 @@ export function getPayTransaction(data: SuiTransactionKind): Pay | undefined {
   return 'Pay' in data ? data.Pay : undefined;
 }
 
+export function getPaySuiTransaction(
```

Comment:
You should also add a function for `PayAllSui`

Reply:
Done.